### ISSUE TITLE
Base URL option

### ISSUE THREAD

**Issue by reinink** [thephpleague/glide#13]
I think it would be wise to have a base URL option when configuration the server. Quite often image routes are appended with `/img/` or something similar when used within an app.

One option would be to simply modify the request path before outputting the image. For example:

```php
$baseUrl = '/img';
$path = substr($request->getPathInfo(), 0, strlen($baseUrl));

$glide->outputImage($path, $request->query->all());
```

However, since this is likely to be a very common task, it might be good to offer this functionality with the package. Something like this:

```php
$glide = Server::create([
    // other config
    'base_url' => '/img',
]);

$glide->outputImage($request);
```

Basically the `base_url` would be omitted when finding the source file. If a more complicated base url manipulation is required, it can still be done manually by simply omitting the `base_url` option.

Thoughts?

**Comment by reinink** [comment]
Done d4a22b37eea1698218e85122583946786367f71d
